content/applications/sales/subscriptions/configuration/subscription_products.rst

```diff
@@ -1,54 +1,74 @@
-=========================
-Add subscription products
-=========================
+===============================
+Configure subscription products
+===============================
 
-To properly sell your subscriptions using our amazing **Odoo Subscriptions** application, you must
-follow these steps:
+By integrating closely with the Odoo *Sales* app, the *Subscriptions* app enables users to sell
+subscription products alongside regular sales products. While regular products are sold on a
+one-time basis, subscription products are sold on a renewing basis, generating recurring revenue.
 
-1. **Create your own subscription templates**
-2. **Create your own subscription products with the right settings**
+Configure recurrence periods
+============================
 
-As a result, you will be able to manage your subscriptions like any other product, create your
-quotations and continue the sales flow to track the number of subscriptions you sell and manage the
-revenue they generate.
+To get started with subscriptions, first make sure that the *recurrence periods* are configured as
+needed.
 
-Configuration
-=============
+Recurrence periods are the time periods in which subscriptions renew. They designate how often
+the customer pays for (and receives) subscription products.
 
-Go to :menuselection:`Subscriptions --> Subscription products` to learn how you can configure your
-own subscription products. You have the possibility to create a new product or edit an existing one.
-Once named, be careful to select the option *Can be sold* and deselect *Can be purchased*. For the
-product type, it is recommended to use *Service* for subscription products as they are non-material
-products that you provide to your customers. Finally, you can adapt your prices and also add an
-internal reference.
+To configure recurrence periods, go to :menuselection:`Subscriptions --> Configuration -->
+Recurrence Periods`.
 
-.. image:: subscription_products/subscription-products-configuration.png
-  :align: center
-  :alt: View of a subscription product form in Odoo Subscriptions
+The *Subscriptions* app comes with some basic recurrence periods already configured. These can be
```

Comment:
It might be good to mention the specific recurrence periods that come pre-configured.